specs-go/config.go

```diff
@@ -117,6 +117,11 @@ type Mount struct {
 	Source string `json:"source,omitempty"`
 	// Options are fstab style mount options.
 	Options []string `json:"options,omitempty"`
+
+	// UID/GID mappings used for changing file owners w/o calling chown, fs should support it.
+	// Every mount point could have its own mapping
+	UIDMappings []LinuxIDMapping `json:"uid_mappings,omitempty"`
+	GIDMappings []LinuxIDMapping `json:"gid_mappings,omitempty"`
```

Comment:
Please add `platform:"linux"` as suggested by @giuseppe (similar to how it is done for `Type` above).